internal/osutil/osutil.go

```diff
@@ -210,3 +213,8 @@ func init() {
 func IsWindowsExecutable(path string) bool {
 	return execExts[strings.ToLower(filepath.Ext(path))]
 }
+
+func FreeDiskSpaceMB(path string) (uint64, error) {
```

Comment:
I suggest `int64` and just leaving it as bytes here. (We don't use unsigned ints for arithmetic, for safety reasons.)